chess-AI/checkpointing.py

```diff
@@ -0,0 +1,19 @@
+import torch
```

Comment:
Can you name this file `io.py` instead? That way, if you want to do any logging or other stuff later, it can all be contained in this file.

Reply:
It will also let us combine AI logging, game loop logging, etc in one file when we get further down the road.

Reply:
> It will also let us combine AI logging, game loop logging, etc in one file when we get further down the road.

io is already an existing standard python library name so I'm not sure how I would import the functions, but I can change the name to be ai_io which is similar.

Reply:
Ah ok you can do that or leave as checkpointing.py whatever u prefer